geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/IteratorTrigger.scala

```diff
@@ -94,25 +95,32 @@ object IteratorTrigger extends Logging {
    */
   def useIndexOnlyIterator(ecqlPredicate: Option[Filter],
                            query: Query,
-                           sourceSFT: SimpleFeatureType,
+                           sft: SimpleFeatureType,
                            indexedAttribute: Option[String] = None): Boolean = {
+    if (useDensityIterator(query: Query)) {
```

Comment:
Oh, yeah I didn't mean to do that -  I'll fix it. It is technically valid, I guess, but you're right it looks weird.